index.html

```diff
@@ -0,0 +1,109 @@
+    <meta name="viewport" property="og:title" content="width=device-width">
+    <title>Website</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+  <main id="main-doc">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+        <ul>
+        <li><header> How to work with HTML , CSS, javascript</header></li>
+        <li><a class="nav-link" href="#introduction">Introduction</a></li>
+        <li><a class="nav-link" href="#prerequisite">prerequisite</a></li>
+        <li><a class="nav-link" href="#html">html</a></li>
+        <li><a class="nav-link" href="#CSS">CSS</a></li>
+        <li><a class="nav-link" href="#javascript">javascript</a></li>
+    </ul>
+    </nav>
+    <br>
+      <section class="main-section" id="introduction">
+        <header>Introduction</header>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint <code>tempore sequi</code> distinctio enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt recusandae eius vitae.</p>
+        <ul>
+          <li>coding</li>
+          <li>programming</li>
+        </ul>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+          culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio
+          enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+          recusandae eius vitae.</p>
+
+      </section>
+      <br>
+      <section class="main-section" id="prerequisite">
+        <header>prerequisite</header>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+          culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio <code>tempore sequi</code>
+          enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+          recusandae eius vitae.</p>
+          <ul>
+            <li>basic GIT</li>
+            <li> VScode studio </li>
+          </ul>
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut dolores molestiae unde debitis, rerum voluptatem! Ut
+            culpa porro dolores dolore accusamus eum quas doloribus repellat assumenda. Perspiciatis consequatur sint distinctio
+            enim, ea tempore sequi veniam et id, vero eligendi perferendis ex deserunt, debitis ut autem. Tenetur incidunt
+            recusandae eius vitae.</p>
+          
+    </section>
+    <br>
+    <section class="main-section" id="html">
+        <header>html</header>
```

Comment:
use `h2` tag instead of header

Reply:
* Kindly explain why you request a change - it might be obvious for you but not for someone else
* Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it ❓

Reply:
please use the h2 tag instead of header, just remember from HTML and CSS good practices h2 are used to highlight important pieces of information and head up subsections on your page